code/src/App.js

```diff
@@ -1,9 +1,85 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+import { NewMessageForm } from './components/NewMessageForm'
+import { Messages} from './components/Messages'
+
+import { API_URL, LIKE_API } from './reusables/urls'
 
 export const App = () => {
+
+  const [messageList, setMessageList] = useState([])
+  const [newMessage, setNewMessage] = useState('')
+
+  useEffect(() => {
+    fetchList()
+  }, []);
+
+  const fetchList = () => {
+    fetch(API_URL)
+      .then(res => res.json())
+      .then(messages => setMessageList(messages))
+      .catch(error => console.error(error))
```

Comment:
All console outputs always need to be removed before moving your code to production.